composer.json

```diff
@@ -12,40 +12,39 @@
     "require" : {
         "php": "^5.6 || ^7.0",
         "cocur/slugify": "^1.4 || ^2.5",
-        "friendsofsymfony/comment-bundle": "^2.0.13",
```

Comment:
Can we safely remove this one? Is it unused on the ecommerce?

Reply:
It is already required here https://github.com/sonata-project/SonataCommentBundle/blob/3.x/composer.json#L21

Reply:
In this case I did a fast search and couldnt find code that uses this bundle directly